Makefile

```
@@ -1,25 +1,84 @@
-TERRAFORM      = $(DOCKER_COMPOSE) terraform
+DOCKER_COMPOSE    = docker compose
+COMPOSE_ENV_FLAG  = $(if $(EFFECTIVE_ENV_FILE),--env-file $(EFFECTIVE_ENV_FILE),)
+COMPOSE           = $(DOCKER_COMPOSE) $(COMPOSE_ENV_FLAG)
 
 # Misc
-.DEFAULT_GOAL  = help
-.RECIPEPREFIX  +=
-.PHONY: $(filter-out vendor node_modules,$(MAKECMDGOALS))
+.DEFAULT_GOAL     = help
+.RECIPEPREFIX    +=
+.PHONY: help start pulumi-preview pulumi-up pulumi-refresh pulumi-destroy \
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)